README.md

```diff
@@ -43,6 +31,36 @@ pnpm --filter "./packages/**" build
 pnpm --filter "./packages/**" test
 ```
 
+## Contributing
+
+### Assign yourself to an issue
+Read through the existing [issues](https://github.com/OpenFn/adaptors/issues) and assign yourself to the issue you have chosen. If you need clarification, don't hesitate leave a comment on the issue and we will get back to you as soon as possible. 
+
+If there isn't already an issue for the feature you would like to contribute, please make one and assign yourself.
+
+
+### Open a pull request
+
+1. Clone the adaptors repository, then [fork it](https://docs.github.com/en/get-started/quickstart/fork-a-repo). 
+
+2. Make your changes by following the [working with adaptors](#working-with-adaptors) section. 
+
+3. Open a draft PR, fill out the pull request template (this will be added automatically for you) then make sure to self-review your code and go through the 'Review checklist'. Leave any notes for the reviewer in the 'Details' section.
+
+4. Mark the PR as ready for review, and assign @stu or @taylor.
+
+### Working with adaptors
+
+1. Creating a new adaptor
+Create a copy of the adaptor [template](https://github.com/OpenFn/adaptors/tree/dab11c0845c1affb01e822ed44494906dde1fc09/packages/template), and rename it.  
```

Comment:
I would suggest that a local path, rather than a link to github, is more helpful here:
```
Copy and rename the adaptor template at `packages/template` and rename it.  
```